references.bib

```diff
@@ -433,7 +486,8 @@ @article{1929Bethe
 url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/andp.19293950202},
 xxxeprint = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/andp.19293950202},
 abstract = {Abstract Der Einfluß eines elektrischen Feldes von vorgegebener Symmetrie (Kristallfeld) auf ein Atom wird wellenmechanisch behandelt. Die Terme des Atoms spalten auf in einer Weise, die von der Symmetrie des Feldes und vom Drehimpuls l (bzw. j) des Atoms abhängt. s-Terme spalten allgemein, p-Terme in Feldern von kubischer Symmetrie nicht auf. Für den Fall, daß die einzelnen Elektronen des Atoms separat behandelt werden dürfen (aufgehobene Wechselwirkung im Atom), werden zu jedem Term im Kristall die Eigenfunktionen nullter Näherung angegeben; aus diesen ergibt sich eine für den Term charakteristische Gruppierung der Elektronendichte nach den Symmetrieachsen des Kristalls. — Die Größe der Termaufspaltung bewegt sich in der Ordnung einiger hundert cm−1. — Bei tetragonaler Symmetrie kann ein quantitatives Maß für die Abweichung von der kubischen Symmetrie definiert werden, welches eindeutig die stabilste Elektronenanordnung im Kristall bestimmt.},
-year = {1929}
+year = {1929},
+url={},
```

Comment:
Why is this `url` here when there's already a `url` above it?

Reply:
This has been removed.